metadata/payments_metadata/meta/merged.xml

```diff
@@ -259,30 +266,12 @@
                 <externally-managed-flag>false</externally-managed-flag>
                 <default-value>false</default-value>
             </attribute-definition>
-            <attribute-definition attribute-id="Cybersource_CruiseApiKey">

```

Comment:
Ya'll removed these `cruise` definitions from the system-objects (ie here)
You removed them from `cartridges/int_cybs_sfra_base/cartridge/configuration/preferences/customPreferences.js`

 but did not update:
`cartridges/int_cybs_sfra_base/cartridge/configuration/index.js`
which still has code like:
        `payerAuthenticationCruiseAPIKey: config.payerAuthenticationCruiseAPIKey || customPreferences.PayerAuthentication.Preferences.CruiseAPIKey.getValue(),`

And that's causing some null pointer issues on a clean system

A system w/ a previously installed version of this cartridge will work because this system object definition will still be there